### PR TITLE
Update some outdated license headers for 1.7.6.x

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Update some outdated license headers for 1.7.6.x
| Type?         | refacto
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | Travis should be green

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/15092)

### CODE REVIEW

File: admin-dev/themes/new-theme/tests/cldr/number-symbol.spec.js [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
```
Comment (matks) [inline admin-dev/themes/new-theme/tests/cldr/number-symbol.spec.js:24]: I dont know if we should put license headers on JS tests files 🤔 we do it for php files, but the goal of license headers is to be shipped in the Zip Archive, right ? so these tests are fine

By the way @eternoendless we should consequently ignore these files in the Release Builder, right ?

**Comment by matks** [comment]
Done using SF command `./bin/console prestashop:licenses:update`

File: admin-dev/themes/new-theme/tests/cldr/number-symbol.spec.js (hunk 1 above)
Comment (eternoendless) [reply admin-dev/themes/new-theme/tests/cldr/number-symbol.spec.js:24]: > I dont know if we should put license headers on JS tests files

People can still pick up our files from Github so I think we should put the header on ALL files.

> we should consequently ignore these files in the Release Builder, right ?

Yes we should

File: src/PrestaShopBundle/Translation/Provider/TranslationFinder.php [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * 2007-2019 PrestaShop SA and Contributors
+ * 2007-2019 PrestaShop and Contributors
```
Comment (eternoendless) [inline src/PrestaShopBundle/Translation/Provider/TranslationFinder.php:4]: It's "PrestaShop SA and Contributors" ([see devdocs](https://devdocs.prestashop.com/1.7/development/coding-standards/#license-information))

**Review by eternoendless — CHANGES_REQUESTED** [review]

File: src/PrestaShopBundle/Translation/Provider/TranslationFinder.php (hunk 2 above)
Comment (matks) [reply src/PrestaShopBundle/Translation/Provider/TranslationFinder.php:4]: This is not what is being computed by the UpdateLicensesCommand
https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Command/UpdateLicensesCommand.php#L40

Are you sure the devdocs is right and the code is wrong ? If yes, we need to update all license headers for both 1.7.6.x and develop 😞

Comment (eternoendless) [reply src/PrestaShopBundle/Translation/Provider/TranslationFinder.php:4]: Yep I'm positive

Comment (matks) [reply src/PrestaShopBundle/Translation/Provider/TranslationFinder.php:4]: Issue created https://github.com/PrestaShop/PrestaShop/issues/15116

**Comment by eternoendless** [comment]
Replaced by #15125

**Pull request closed without merging**